README.md

```diff
@@ -183,14 +183,14 @@ All APIs and Models' documentation can be found at the [Generated client's docs]
 
 ## Compatibility
 
-|  client version  | 1.12      | 1.13     | 1.14     |  1.15     |  1.16    |  1.17    |
+|  client version  | 1.13      | 1.14     | 1.15     |  1.16     |  1.17    |  1.18    |
 |------------------|-----------|----------|----------|-----------|----------|----------|
-|  4.0.0           |  ✓        |  -       |  -       | x         | x        | x        |
-|  5.0.0           |  +        |  ✓       |  -       | -         | x        | x        |
-|  6.0.1           |  +        |  +       |  ✓       | -         | -        | x        |
-|  7.0.0           |  +        |  +       |  +       | ✓         | -        | -        |
-|  8.0.2           |  +        |  +       |  +       | +         | ✓        | -        |
-|  9.0.0           |  +        |  +       |  +       | +         | +        | ✓        |
+|  5.0.0           |  ✓        |  -       |  -       | x         | x        | x        |
+|  6.0.1           |  +        |  ✓       |  -       | -         | x        | x        |
+|  7.0.0           |  +        |  +       |  ✓       | -         | -        | x        |
+|  8.0.2           |  +        |  +       |  +       | ✓         | -        | -        |
+|  9.0.2           |  +        |  +       |  +       | +         | ✓        | -        |
+|  10.0.0           |  +        |  +       |  +       | +         | +        | ✓        |
```

Comment:
(if my browser is displaying correct) here's an extra space, is it?

Reply:
fixed. (it was because `10.0.0` has one extra digit :)